apps/vmq_plugin/priv/vmq_plugin.schema

```diff
@@ -18,3 +18,14 @@
          end
  end}.
 
+{translation, "vmq_plugin.priority",
```

Comment:
How is this `vmq_plugin.priority` config used at present?

Reply:
`plugins.mypluginname.priority = 5`

Reply:
priority set in this way^ in the config the `apps/vmq_server/priv/vmq_server.schema` will sort them according to the priority and return a list and then `apps/vmq_server/src/vmq_server_app.erl` will load the plugins in order of that list on vernemq startup.